.cirrus.yml

```diff
@@ -41,16 +42,15 @@ check_android_task:
     curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
     chmod +x ./cc-test-reporter
     ./cc-test-reporter before-build
-  screen_record_background_script:
-    for n in $(seq 1 20); do adb exec-out screenrecord --time-limit=180 --output-format=h264 - > $n.h264; done
-  check_script:
-    ./gradlew check connectedCheck
+  screen_record_background_script: for n in $(seq 1 20); do adb exec-out screenrecord --time-limit=180 --output-format=h264 - > $n.h264; done
```

Comment:
Same comment regarding the syntax for multiple commands.
It might not work in this case due to the loop. I let you check.

Reply:
couldn't unrol much because of the loop, but still changed the syntax a bit to make it more coherent